scripts/core/EXT.rst

```diff
@@ -42,9 +69,9 @@ The following naming conventions must be followed for **standard** extensions:
 
 ## --validate=off
   - All extension functions must be postfixed with `Ext`
-  - All macros must use all caps `${X}_NAME_EXT` convention
-  - All structures, enumerations and other types must follow `${x}_name_ext_t` snake case convention
-  - All enumerator values must use all caps `${X}_ENUM_EXT_ETOR_NAME` convention
+  - All macros must use all caps `${X}_NAME_EXT` or `${S}_NAME_EXT` convention
```

Comment:
"All macros must use all caps *\<PREFEX\>*_NAME_EXT convention, where **PREFIX** is appropriately applied for the macro namespace, i.e. `${X}_NAME_EXT`, `${S}_NAME_EXT` or `${T}_NAME_EXT`."

Reply:
@wdamon-intel fixed this now, please check